index.html

```diff
@@ -215,32 +215,64 @@ <h3>Exposing HTML Features That Do Not Directly Map to Accessibility APIs</h3>
       </p>
     </section>
     <section>
-      <h3>HTML Element Role Mappings</h3>
+      <h3 id="minimum-role">Exposing HTML Features That Require a Minimum Role</h3>
+      <p>
+        A <dfn>minimum role</dfn> is the equivalent WAI-ARIA role an element will map to if the element does not have a more specific implicit role, e.g., a non-generic role. This can help ensure that users of assistive technologies get the best possible experience for commonly-used and valid HTML markup where otherwise a role would not be exposed.
+      </p>
+      <p>
+        A minimum role is provided when all of the following conditions are true:
+      </p>
       <ul>
-        <li>HTML elements with implicit WAI-ARIA role semantics MUST be mapped to platform <a class="termref">accessibility APIs</a> according to the identified WAI-ARIA role mapping as defined in the [[core-aam-1.2]] specification.</li>
-        <li>"Not mapped" means the element does not need to be exposed via an <a class="termref">accessibility API</a>. This is usually because the element is not displayed as part of the user interface. However, authors can force some of these elements to be rendered. For instance, by overriding user agent styles to render elements that would have been otherwise set to `display: none`. In these cases, the user agent SHOULD map such elements to the role of <a class="core-mapping" href="#role-map-generic">`generic`</a>.</li>
-        <li>Where applicable, how an element participates in the computation of its own or another element's <a data-cite="accname-1.2/#dfn-accessible-name">accessible name</a> and/or <a data-cite="accname-1.2/#dfn-accessible-description">accessible description</a> is described in the <a href="#accessible-name-and-description-computation">Accessible Name and Description Computation</a> section of this document.</li>
-        <li>Where an element is indicated as having &quot;No corresponding (WAI-ARIA) role&quot;, or is mapped to the <a class="core-mapping" href="#role-map-generic">`generic`</a> role, user agents MUST NOT expose the <a class="core-mapping" href="#ariaRoleDescription">`aria-roledescription`</a> property value in the <a class="termref">accessibility tree</a> unless the element has an explicit, conforming `role` attribute value which [[WAI-ARIA-1.2]] does not prohibit the use of `aria-roledescription`.</li>
         <li>
-          <strong>IAccessible2:</strong>
-          <ul>
-            <li>All elements with accessible objects should implement the IAccessible, IAccessible2 and IAccessible2_2 interfaces.</li>
-          </ul>
+          the element either has no implicit mapping or has an implicit `generic` role mapping, 
```

Comment:
```suggestion
          the element either has no implicit mapping or has an implicit `generic` role mapping, 
```

for this line, I'm looking for suggestions on how to call out that platform-specific roles would also remain.  e.g., a video has it's own platform role, but not a specific ARIA role.  @cookiecrook ideas?

Reply:
From the discussion today, maybe make this a role selection algorithm, such as:

1. explicit role value e.g. role="switch checkbox" (note: in earlier ARIA discussions, checkbox is the "fallback role" if switch is not implemented in the browser, though I don't recall if we codified that term anywhere)
2. Implicit role (e.g. from HTML-AAM) or native implicit implementation role (like `<video>`)
3. If current "role" from above steps is none or generic, use "minimum implicit role" if defined in AAM.
4. no role? ignored.